configuration/jsonkeyvalues/jsonkeyvalues.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
@reagan-meant , could you save the config in a dedicated file?
Maybe `vaccinationReportConfig.json`

Reply:
Done